src/components/CandidateBox.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import Candidate from './Candidate';
+import axios from 'axios';
+import { url } from './url';
+
+const CandidateBox = () => {
+  const [voteCandidate, setVoteCandidate] = useState('');
+  const getCandidates = async () => {
+    try {
+      const Candidates = await axios.get(`${url}/candidates`);
+      setVoteCandidate(Candidates.data);
```

Comment:
1. 이렇게 `data`를 바로 뽑아낼 수도 있어요!

2. `/candidates`에서 후보자리스트를 받아서 `state`에 저장하는것이니 `setCandidateList`와 같은 네이밍이 더 적절할 것 같습니다!

```suggestion
    try {
      const {data} = await axios.get(`${url}/candidates`);
      setCandidateList(data);
```

Reply:
앗 그러네요!! 저렇게 바로 뽑는게 더 효율적이네요! 👍